requirements.txt

```diff
@@ -1,6 +1,6 @@
 # requirements for most basic library use
 astropy>=2.0.3,<3.0.0; python_version <= '2.7'
-astropy>=2.0.3; python_version >= '3.0'
+astropy>=2.0.3!=4.2.1,!=4.0.5; python_version >= '3.0'
```

Comment:
Is a comma needed between `2.0.3` and `!=`?

Reply:
Yes, see updated commit.